README.md

```diff
@@ -38,7 +38,13 @@ $ git hist
 
 ![Screenshot](http://i.imgur.com/D5AqNUz.png)
 
-*If your terminal displays weird characters, please make sure you're running the latest version of `less` before reporting an issue. Thanks! :beers:*
+*If your terminal displays weird characters, please make sure you're running at least version `471` of `less` before reporting an issue. Thanks! :beers:*
+
+*You can install `less` via [Homebrew](http://brew.sh):*
```

Comment:
What do you think if we mention that this command will actually install a "dev" version of `less`? Something like:

> Please note that this version of less is still in beta. If you use [Homebrew](http://brew.sh), you'll need a special command to install it: